Cargo.toml

```diff
@@ -8,18 +8,17 @@ edition = "2018"
 crate-type = ["cdylib"]
 
 [features]
-dev = ["console_log", "console_error_panic_hook"]
+dev = ["console_error_panic_hook"]
 
 [dependencies]
-console_error_panic_hook = { version = "0.1", optional = true }
-console_log = { version = "0.1", optional = true }
-log = { version = "0.4", features = ["release_max_level_warn"] }
-serde = { version = "1.0", features = ["derive"] }
-serde_repr = "0.1"
-serde-wasm-bindgen = "0.1"
-stacker = "= 0.1.5" # last known working version, 0.1.6 breaks wasm support
-wasm-bindgen = "0.2.54"
+console_error_panic_hook = { version = "0.1.6", optional = true }
+log = { version = "0.4.14", features = ["release_max_level_warn"] }
+serde = { version = "1.0.125", features = ["derive"] }
+serde_repr = "0.1.6"
+serde-wasm-bindgen = "0.1.3"
+stacker = "0.1.13"
+wasm-bindgen = "0.2.72"
+wasm-bindgen-rayon = "1.0.1"
 
-ra_ide_api = { git = "https://github.com/rust-analyzer/rust-analyzer", features = ["wasm"] }
-ra_syntax =  { git = "https://github.com/rust-analyzer/rust-analyzer" }
-ra_text_edit =  { git = "https://github.com/rust-analyzer/rust-analyzer" }
+ide = { git = "https://github.com/rust-analyzer/rust-analyzer", rev = "27befe6c7" }
+ide_db = { git = "https://github.com/rust-analyzer/rust-analyzer", rev = "27befe6c7" }
```

Comment:
Can/should we used auto-published crates here?

Reply:
Sure, and it works 🎉